Decisions/knowledge-management/information-classification.md

```diff
@@ -0,0 +1,54 @@
+# Information Classification Levels
+
+| Status      | draft                                            |
+| :---------- | :-------------------------------------------------- |
+| **PR #**    | [16](https://github.com/blindnet-io/openness/pull/16) |
+| **Sponsor** | Filip Radulovic (filip@blindnet.io)                        |
+
+## (-2) Public
+
+Promoted information
+
+Examples:
+
+- blindnet blog post
+- any information on blindnet.io
+- any content referenced by a link in blindnet.io
+- any content shared on social media
+
+## (-1) Open
+
+Information anyone could access online via an URL.
+
+Example:
+
+- unpromoted Github repository
+- unreferenced video on Youtube
+- Google document shared with "anyone with the link"
+
+## (0) _Unclassified_
+
+> Default level
+
+Information accessible without extensive efforts.
+
+- **potential effect**: same as restricted (should be confined to the person who recorded the information)
+- **maximum duration**: expected minimal
+- **minimum requirements**: simple authentication
+
+Examples:
+
+- personnal notes in evernote
+- video recorded on a personnal smartphone
+
+## (1) Restricted
```

Comment:
Yeah, we'll need to define "damage" and "undesirable effects" more clearly here. "Undesirable effects" basically mean "yeah, that's bad, but we can fix it" (think: someone said to one of your friends you're friend with one of their enemies). "Damage" is more "OMG what did happen we'll never recover from that" :smile: - fixing it would take tremendous efforts, or may not even be possible (think: someone stole your money from your bank account).